lib/chef/user.rb

```diff
@@ -144,12 +287,17 @@ class << self
     end
 
     def self.list(inflate=false)
-      response = Chef::REST.new(Chef::Config[:chef_server_url]).get_rest('users')
-      users = if response.is_a?(Array)
```

Comment:
All `knife user` commands are now backwards compatible, see https://github.com/chef/chef/commit/768ef158fc6666dddf2fea7f8fda4613f5b31ab7 and the PR description for details.